### PR TITLE
Remove periodic commit

### PR DESCRIPTION
**Pull request by hedengran**
~~Builds on top of #1453~~

### CODE REVIEW

**Comment by hedengran** [comment]
[Remove Periodic commit from Documentation repo](https://trello.com/c/P5QdHkGA/979-remove-periodic-commit-from)

File: cypher/cypher-docs/src/docs/dev/ql/load-csv/index.asciidoc [hunk 1]
```diff
@@ -87,8 +87,6 @@ include::import-data-from-a-csv-file-with-a-custom-field-delimiter.asciidoc[]
 
 include::importing-large-amounts-of-data.asciidoc[]
 
-include::setting-the-rate-of-periodic-commits.asciidoc[]
```
Comment (tobias-johansson) [inline cypher/cypher-docs/src/docs/dev/ql/load-csv/index.asciidoc:90]: Is this just gone or should we import a replacement here?

**Review by tobias-johansson — COMMENTED** [review]
Nice work! Just a question about an import

File: cypher/cypher-docs/src/docs/dev/ql/load-csv/index.asciidoc (hunk 1 above)
Comment (hedengran) [reply cypher/cypher-docs/src/docs/dev/ql/load-csv/index.asciidoc:90]: Oh, nice catch. This should be replaced, thank you.

**Review by tobias-johansson — APPROVED** [review]
:+1:

File: cypher/cypher-docs/src/docs/dev/general/query-transactions.asciidoc [hunk 2]
```diff
@@ -30,7 +30,7 @@ Transactions can be either _explicit_ or _implicit_.
   * Can execute a single Cypher query.
   * Are committed automatically when the query finishes successfully.
 
-Queries that start separate transactions themselves, such as queries using <<subquery-call-in-transactions, `CALL { ... } IN TRANSACTIONS`>> or <<query-using-periodic-commit-hint, `PERIODIC COMMIT`>> are only allowed in _implicit_ mode.
+Queries that start separate transactions themselves, such as queries using <<subquery-call-in-transactions, `CALL { ... } IN TRANSACTIONS`>>, are only allowed in _implicit_ mode.
 
 For examples of the API's used to start and commit transactions, refer to the API specific documentation:
```
Comment (renetapopova) [inline cypher/cypher-docs/src/docs/dev/general/query-transactions.asciidoc]:
```suggestion
For examples of the APIs used to start and commit transactions, refer to the API specific documentation:
```

File: cypher/cypher-docs/src/docs/graphgists/import/import-csv-with-cypher.asciidoc [hunk 3]
```diff
@@ -105,16 +105,18 @@ CREATE CONSTRAINT movie FOR (movie:Movie) REQUIRE movie.id IS UNIQUE
 
 Now importing the relationships is a matter of finding the nodes and then creating relationships between them.
 
-For this query we'll use `USING PERIODIC COMMIT` (see <<cypher-manual#query-using-periodic-commit-hint>>) which is helpful for queries that operate on large CSV files.
-This hint tells Neo4j that the query might build up inordinate amounts of transaction state, and so needs to be periodically committed.
-In this case we also set the limit to `500` rows per commit.
+For this query we'll use `CALL +++{ ... }+++ IN TRANSACTIONS` (see <<cypher-manual#subquery-call-in-transactions>>) which is helpful for queries that operate on large CSV files.
+This is a way to periodically commit progress in a new transaction.
+In this case we also set the limit to `500` rows per transaction.
```
Comment (renetapopova) [inline cypher/cypher-docs/src/docs/graphgists/import/import-csv-with-cypher.asciidoc]:
```suggestion
In this case, you also set the limit to `500` rows per transaction.
```

File: cypher/cypher-docs/src/docs/dev/ql/load-csv/index.asciidoc [hunk 4]
```diff
@@ -28,7 +28,7 @@
   Alternatively, `LOAD CSV` also supports accessing CSV files via _HTTPS_, _HTTP_, and _FTP_.
 * `LOAD CSV` supports resources compressed with _gzip_ and _Deflate_. Additionally `LOAD CSV` supports locally stored CSV files compressed with _ZIP_.
 * `LOAD CSV` will follow _HTTP_ redirects but for security reasons it will not follow redirects that changes the protocol, for example if the redirect is going from _HTTPS_ to _HTTP_.
-* `LOAD CSV` is often used in conjunction with the query hint `PERIODIC COMMIT`; more information on this may be found in <<query-using-periodic-commit-hint>>.
+* `LOAD CSV` is often used in conjunction with the query clause `CALL +++{ ... }+++ TRANSACTIONS`. More information on this may be found in <<subquery-call-in-transactions, CALL +++{ ... }+++ IN TRANSACTIONS>>.
```
Comment (renetapopova) [inline cypher/cypher-docs/src/docs/dev/ql/load-csv/index.asciidoc]:
```suggestion
* `LOAD CSV` is often used in conjunction with the query clause `CALL +++{ ... }+++ TRANSACTIONS`. For more information, see <<subquery-call-in-transactions, CALL +++{ ... }+++ IN TRANSACTIONS>>.
```

File: cypher/cypher-docs/src/docs/graphgists/import/import-csv-with-cypher.asciidoc [hunk 5]
```diff
@@ -105,16 +105,18 @@ CREATE CONSTRAINT movie FOR (movie:Movie) REQUIRE movie.id IS UNIQUE
 
 Now importing the relationships is a matter of finding the nodes and then creating relationships between them.
 
-For this query we'll use `USING PERIODIC COMMIT` (see <<cypher-manual#query-using-periodic-commit-hint>>) which is helpful for queries that operate on large CSV files.
-This hint tells Neo4j that the query might build up inordinate amounts of transaction state, and so needs to be periodically committed.
-In this case we also set the limit to `500` rows per commit.
+For this query we'll use `CALL +++{ ... }+++ IN TRANSACTIONS` (see <<cypher-manual#subquery-call-in-transactions>>) which is helpful for queries that operate on large CSV files.
```
Comment (renetapopova) [inline cypher/cypher-docs/src/docs/graphgists/import/import-csv-with-cypher.asciidoc]:
```suggestion
For this query, we will use `CALL +++{ ... }+++ IN TRANSACTIONS` (see <<cypher-manual#subquery-call-in-transactions>>) to periodically commit progress in a new transaction. 
This is helpful for queries that operate on large CSV files.
```

File: cypher/cypher-docs/src/docs/graphgists/import/import-csv-with-cypher.asciidoc [hunk 6]
```diff
@@ -105,16 +105,18 @@ CREATE CONSTRAINT movie FOR (movie:Movie) REQUIRE movie.id IS UNIQUE
...
+This is a way to periodically commit progress in a new transaction.
```
Comment (renetapopova) [inline cypher/cypher-docs/src/docs/graphgists/import/import-csv-with-cypher.asciidoc]:
```suggestion

```

**Review by renetapopova — COMMENTED** [review]
A few editorial suggestions.

**Review by renetapopova — APPROVED** [review]
LGTM, thanks @hedengran!

**Pull request merged**